src/app/CMakeLists.txt

```diff
@@ -36,6 +41,7 @@ if (ANDROID)
     ${RESOURCES}
     ${CMAKE_SOURCE_DIR}/images/images.qrc
     ${CMAKE_SOURCE_DIR}/resources/resources.qrc
+    ${ICON_SRC}
```

Comment:
@m-kuhn , is that meant to reside within the if (ANDROID) condition?

Reply:
that's why it's not working 😆